src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -131,12 +129,12 @@ class SpatialPooler : public Serializable
         pools of all columns). The inhibition logic will insure that at
         most N columns remain ON within a local inhibition area, where
         N = localAreaDensity * (total number of columns in inhibition
-        area). If localAreaDensity is set to a negative value output
+        area). If localAreaDensity is set to 0, output
```

Comment:
Are you sure 0 means disable and not -1?

Reply:
yep, these 2 caused trouble, I don't know the API (?) but in the code it is set this way. Or should I do it otherway and change code to -1? (..that would be problematic, as we use UInt)

Reply:
I think you should check the API.  I think -1 indicated the value was disabled or the feature not used.  I know this is the case with something, but not sure this is the variable.  If so, it means the value should not be UInt.